domainic-attributer/docs/USAGE.md

```
@@ -0,0 +1,723 @@
+# Domainic::Attributer Usage Guide
+
+A comprehensive guide to all features and capabilities of Domainic::Attributer.
+
+* [Core Concepts](#core-concepts)
+   * [Arguments vs Options](#arguments-vs-options)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 3
Unordered list indentation

(MD007, ul-indent)